### PR TITLE
QtCreator IDE with inline ROOT help

### PR DESCRIPTION
**Pull request by ferdymercury**
As suggested by @couet, I am preparing a blog post on how to use QtCreator's IDE for building ROOT, or ROOT-depending CMake projects, with in-built help support using the "F1" button.

This is still a draft.

Here a list of "documentation" issues (that I found while working on this):

- [x] (Main stopper) https://github.com/doxygen/doxygen/issues/9094
- [x]  https://github.com/doxygen/doxygen/issues/9056
- [ ] https://github.com/doxygen/doxygen/issues/9040
- [ ] https://github.com/doxygen/doxygen/issues/9135
- [x] https://github.com/doxygen/doxygen/issues/9138
- [x] https://github.com/doxygen/doxygen/issues/9142
- [ ] https://github.com/doxygen/doxygen/issues/9143
- [x] https://github.com/doxygen/doxygen/issues/9144
- [ ] https://github.com/doxygen/doxygen/issues/9148
- [ ] https://github.com/doxygen/doxygen/issues/9155
- [ ] https://github.com/doxygen/doxygen/issues/9157
- [ ] https://github.com/doxygen/doxygen/pull/9173
- [ ] https://bugreports.qt.io/projects/QTCREATORBUG/issues/QTCREATORBUG-26862
- [ ] https://bugreports.qt.io/browse/QTBUG-101070
- [x] https://github.com/root-project/root/issues/8781
- [x] https://github.com/root-project/root/pull/9629
- [x] https://github.com/root-project/root/pull/9657
- [ ] https://github.com/root-project/root/pull/9655
- [ ] https://github.com/root-project/root/pull/9654
- [ ] https://github.com/root-project/root/pull/9651
- [x] https://github.com/root-project/root/pull/9943
- [x] https://github.com/root-project/root/pull/9946
- [ ] https://github.com/root-project/root/issues/9953
- [x] https://github.com/root-project/root/pull/9955
- [ ] https://github.com/root-project/root/pull/9966
- [x] https://github.com/root-project/root/pull/9983
- [ ] https://github.com/root-project/root/pull/10000

### CODE REVIEW

**Comment by couet** [comment]
I'll wait your green light to merge it.

**Comment by eguiraud** [comment]
@ferdymercury thanks a lot! @couet please also wait for a review on the content by a ROOT dev to merge this!

**Comment by couet** [comment]
You need to also add you as author in `_data/authors.yml` . You need a picture to.
See for instance "Petr Stepanov"

**Comment by ferdymercury** [comment]
@couet Could you let me know if I sufficiently explain how to run QtCreator to detect doxygen warnings ? Maybe I overlooked a step that seems obvious to me, but not to a new user.

If you can, please let me know if you are able to reproduce something that looks (kind of) this image: https://user-images.githubusercontent.com/10653970/155034554-ca488395-8c6d-4635-ba66-a70c06a262cd.png
Thanks in advance!

**Comment by couet** [comment]
@ferdymercury seems to me it is very clear.
@eguiraud you suggested : "please also wait for a review on the content by a ROOT dev to merge this!" . Have you somebody in mind ? LGTM

**Review by couet — APPROVED** [review]

**Comment by couet** [comment]
@Axel-Naumann Seems to me this new blog looks fine and ready to be posted. What do you think ?

**Comment by couet** [comment]
I just took your post and build it locally It appears like that:
Screenshot 2022-02-22 at 14 09 01
You need a little abstract at the beginning to summarise the post in order it shows nicely on the home page. Also note that if the date of the post is in the future it will not show on the web site. That's may be a good way to keep it not visible until we decide to really post it.

File: blog/_posts/2022-04-01-code-horsepower-f1.md
Comment (couet) [inline blog/_posts/2022-04-01-code-horsepower-f1.md:19]: explictly -> explicitly

Comment (couet) [inline blog/_posts/2022-04-01-code-horsepower-f1.md:19]: concepation -> conception

**Comment by couet** [comment]
Also I do not know if the images are intended to be seen in details, but as its is now it is impossible to read the tex on them.
Example:
Screenshot 2022-02-22 at 14 23 22

**Comment by ferdymercury** [comment]
> Also I do not know if the images are intended to be seen in details, but as its is now it is impossible to read the tex on them

If you click on them, do they get bigger ?

**Comment by couet** [comment]
> Thanks! If you click on them, do they get bigger ?

no, they are static

**Comment by ferdymercury** [comment]
thanks. I commited a fix.

**Comment by couet** [comment]
You start your blog post with:
```
# Coding in ROOT with the horsepower of an F1
```
You should remove it, as it duplicates the text title. Then the text following will be used as abstract on the web site home page.
Example: https://github.com/root-project/web/blob/main/blog/_posts/2014-05-31-rainbow-color-map.md

**Comment by ferdymercury** [comment]
Thanks for the hint. Changed. The paragraph below fits already well as an abstract / introduction.

**Comment by couet** [comment]
@ferdymercury: Do you think it can be merged soon ? I see the qch file here: https://root.cern/reference/ is there.

**Comment by ferdymercury** [comment]
> @ferdymercury: Do you think it can be merged soon ? I see the qch file here: https://root.cern/reference/ is there.

Thanks! That's great.

Yes, I think so, from my side it's ready, though we should wait for Axel's opinion.

It would also be convenient to wait first for these issues to be addressed:

image

Thanks and have a nice weekend!

**Pull request merged**
